src/specs/models/samples-bodies/CreateSampleFile.v2.yaml

```diff
@@ -0,0 +1,23 @@
+title: Create sample file
+description: 'Data required to create a new sample file'
+type: object
+properties:
+  sampleFileId:
+    type: string
+  size:
+    type: number
+  metadata:
+    description: Metadata to append to the s3 object on upload
+    type: object
+    properties:
+      cellrangerVersion:
+        type: string
+        oneOf:
+          - pattern: v2
+          - pattern: v3
+ 
+required:
```

Comment:
why do we not require metadata too?

Reply:
because it is not necessary in most cases, there's only one type of file that requires metadata (the features/genes file)